digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/RoadLinkDAO.scala

```diff
@@ -312,20 +317,32 @@ class RoadLinkDAO {
       getLinksWithFilter(filter(values)).asInstanceOf[Seq[T]]
     } else Seq.empty[T]
   }
-  
- protected def getLinksWithFilter(filter: String): Seq[RoadLinkFetched] = {
-   LogUtils.time(logger,"TEST LOG Getting roadlinks" ){
-     sql"""select linkid, mtkid, mtkhereflip, municipalitycode, shape, adminclass, directiontype, mtkclass, roadname_fi,
+
+  protected def getLinksWithFilter(filter: String): Seq[RoadLinkFetched] = {
+    LogUtils.time(logger,"TEST LOG Getting roadlinks" ){
+      sql"""select linkid, mtkid, mtkhereflip, municipalitycode, shape, adminclass, directiontype, mtkclass, roadname_fi,
                  roadname_se, roadnamesme, roadnamesmn, roadnamesms, roadnumber, roadpartnumber, constructiontype, verticallevel, horizontalaccuracy,
                  verticalaccuracy, created_date, last_edited_date, from_left, to_left, from_right, to_right,
-                 surfacetype, geometrylength
+                 surfacetype, geometrylength, expired_date
           from kgv_roadlink
-          where #$filter and constructiontype in (${ConstructionType.InUse.value},
+          where #$filter
+          and expired_date isnull
+          and constructiontype in (${ConstructionType.InUse.value},
                                                   ${ConstructionType.UnderConstruction.value},
                                                   ${ConstructionType.Planned.value},
                                                   ${ConstructionType.TemporarilyOutOfUse.value})
           """.as[RoadLinkFetched].list
-   }
+    }
+  }
+
+  protected def getExpiredRoadLinks(): Seq[RoadLinkFetched] = {
+    sql"""select linkid, mtkid, mtkhereflip, municipalitycode, shape, adminclass, directiontype, mtkclass, roadname_fi,
+                 roadname_se, roadnamesme, roadnamesmn, roadnamesms, roadnumber, roadpartnumber, constructiontype, verticallevel, horizontalaccuracy,
+                 verticalaccuracy, created_date, last_edited_date, from_left, to_left, from_right, to_right,
+                 surfacetype, geometrylength, expired_date
+          from kgv_roadlink
+          where expired_date notnull
```

Comment:
Muualla koodissa näytetään käyttävän näissä "is not null", mutta jos toimii niin ok.